modules/machine-delete.adoc

```diff
@@ -9,11 +9,10 @@
 
 You can delete a specific machine. 
 
-//todo: confirm if this statement is still accurate w/CPMS
-[NOTE]
+[IMPORTANT]
 ====
-You cannot delete a control plane machine.
-====  
+Do not delete control plane machine unless your cluster uses a control plane machine set that is configured to use the `OnDelete` update strategy.
```

Comment:
Comment got eaten when I updated, but I think this makes sense?

Reply:
OnDelete or RollingUpdate works so I'd drop the part about strategy